x-pack/solutions/security/plugins/security_solution/server/lists_integration/endpoint/validators/endpoint_exceptions_validator.ts

```diff
@@ -50,7 +65,7 @@ export class EndpointExceptionsValidator extends BaseValidator {
     return item;
   }
 
-  async validatePreDeleteItem(): Promise<void> {
+  async validatePreDeleteItem(currentItem: ExceptionListItemSchema): Promise<void> {
     await this.validateHasWritePrivilege();
   }
```

Comment:
nit: the parameter `currentItem` is unnecessary

also, do i understand correctly, that we don't need any additional check here, as `this.validateHasWritePrivilege()` already takes care of it?

Reply:
Good catch. no, I need to add the validation here as well.

Reply:
Also - FYI: in my next PR, I'm going to build up the API integration tests for endpoint exceptions since we don't seem to have many/any at the moement

Reply:
thanks for the FYI - just wanted to ask about it :D